packages/lab/src/Flow/stories/SubFlow/NodeGroup.tsx

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useState } from "react";
+import { css, cx } from "@emotion/css";
+import {
+  NodeResizeControl,
+  NodeResizer,
+  NodeProps as ReactFlowNodeProps,
+} from "reactflow";
+import {
+  HvDropDownMenu,
+  HvListValue,
+  HvTypography,
+  theme,
+} from "@hitachivantara/uikit-react-core";
+import { Fullscreen } from "@hitachivantara/uikit-react-icons";
+import { useHvNode } from "@hitachivantara/uikit-react-lab";
+
+const dashedBorder = `1px dashed ${theme.colors.atmo4}`;
+const containerBorderRadius = "0 16px 16px 16px";
+const minWidth = 300;
+const minHeight = 200;
+
+const classes = {
```

Comment:
We have styles inside the component and then defined here inside `classes`. For consistency, I think we should define all the styles inside `classes` when possible 🙏